.github/dependabot.yml

```diff
@@ -12,8 +12,22 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    reviewers:
+      - tosun-si
+      - lukecwik
+      - kennknowles
+      - robertwb
+      - kileys
+      - apilloud
```

Comment:
Please remove me. (I've mitigated by blocking notifications from beam-starter-* repos.)

Reply:
Thanks for letting us know, removing you on https://github.com/apache/beam-starter-kotlin/pull/13